mesonbuild/modules/pkgconfig.py

```diff
@@ -16,12 +16,87 @@
 from pathlib import PurePath
 
 from .. import build
+from .. import dependencies
 from .. import mesonlib
 from .. import mlog
 from . import ModuleReturnValue
 from . import ExtensionModule
 from ..interpreterbase import permittedKwargs
 
+class DepsHolder:
```

Comment:
Don't call this a holder. It has a very specific meaning in the context of Meson. A holder is always an InterpreterObject that exposes something to the user. Internal helper classes should be differentiated by having a different name.